### PR TITLE
Change SseUtils call sites to call CpuMathUtils instead

### PR DESCRIPTION
**Pull request by briancylui**
After all active SSE intrinsics have been implemented and tested for correctness and speed, this PR asks to make all external `SseUtils` call sites call `CpuMathUtils` instead.  This would allow us to obtain a baseline performance of C# hardware intrinsics APIs in a future PR.

cc: @safern @danmosemsft @eerhardt @tannergooding

### CODE REVIEW

**Comment by briancylui** [comment]
All 5 checks have passed for this new PR submitted today.

After all active SSE intrinsics have been implemented and tested for correctness and speed, this PR asks to make all external `SseUtils` call sites call `CpuMathUtils` instead. This would allow us to obtain a baseline performance of C# hardware intrinsics APIs in a future PR.

This PR is now ready for review.

**Comment by safern** [comment]
It seems like you missed 2 instances:
https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.CpuMath/CpuAligenedMathUtils.cs#L19
https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.CpuMath/CpuAligenedMathUtils.cs#L32

**Comment by briancylui** [comment]
@safern Seems that those two `SseUtils` references are wrapped around by #if and #endif that are only used in Debug.  Thank you for the nice catch - now fixed!

File: src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs [hunk 1]
```diff
@@ -9,6 +9,8 @@ namespace Microsoft.ML.Runtime.Internal.CpuMath
 {
     public static partial class CpuMathUtils
     {
+        public const int CbAlign = 16;
```
Comment (eerhardt) [inline src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs]: This shouldn’t be here.  This is Sse specific, but CpuMathUtils should be agnostic of the ISA.

Comment (eerhardt) [reply src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs]: Maybe if you name it ‘SseCbAlign’, that would work.

Comment (briancylui) [reply src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs]: Agree and thanks!  The only current call site of `CbAlign` is in this file:
https://github.com/dotnet/machinelearning/blob/b51d9f9060acfcbe84405ce37f9c045654573ee0/src/Microsoft.ML.Transforms/RffTransform.cs#L24-L29
where Line 26 should be changed to `using CpuMathUtils;`, and the bunch of `CpuUtils.CbAlign` in subsequent lines should be changed to `CpuMathUtils.SseCbAlign` for the moment.  While this change may expose the ISA-specific element (the `Sse` prefix of `SseCbAlign`) out of `CpuMathUtils`, I think it's good for now.

In the future when we also support AVX, should I change `public const int CbAlign = 16;` to something like the code below?
```
public const int CbAlign = Avx.IsSupported ? 32 : 16;
**OR**
public int GetCbAlign() => Avx.IsSupported ? 32 : 16;
```

**Comment by briancylui** [comment]
test VSTS: public-CI please

**Comment by briancylui** [comment]
test VSTS: public-CI

File: src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs [hunk 2]
```diff
@@ -9,6 +9,8 @@ namespace Microsoft.ML.Runtime.Internal.CpuMath
 {
     public static partial class CpuMathUtils
     {
+        public const int SseCbAlign = 16;
```
Comment (tannergooding) [inline src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs]: What does `CbAlign` stand for (count of bytes alignment)? 

Following .NET Naming Conventions might be desirable...

Comment (briancylui) [reply src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs]: I also want to know what `CbAlign` stands for.  I was following the existing code:
https://github.com/dotnet/machinelearning/blob/b51d9f9060acfcbe84405ce37f9c045654573ee0/src/Microsoft.ML.CpuMath/Sse.cs#L11-L14

Comment (briancylui) [reply src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs]: After some detective work, the convention seems to stem from `AlignedArray.cs`:
https://github.com/dotnet/machinelearning/blob/b51d9f9060acfcbe84405ce37f9c045654573ee0/src/Microsoft.ML.CpuMath/AlignedArray.cs#L19-L28

Comment (tannergooding) [reply src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs]: `Cb` normally stands for something like `count of bytes`.

I would probably rename this to something like `Vector128Alignment`, or something similar. That makes it clear that it is the alignment for the `Vector128<T>` type, and follows a "normal" .NET Naming Convention.

Comment (briancylui) [reply src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs]: Changed the naming of `SseCbAlign` to `Vector128Alignment` in the latest commit.  Thanks for your suggestion!

**Comment by briancylui** [comment]
Incorporated naming suggestions in the latest commits.  All 5 checks have passed with two reviewers' approvals.

**Pull request merged**